src/Libraries/Microsoft.Extensions.Http.Resilience/README.md

```diff
@@ -4,6 +4,11 @@ Resilience mechanisms for `HttpClient` built on the [Polly framework](https://ww
 
 ## Install the package
 
+> [!IMPORTANT]
```

Comment:
@RussKie This is a note helping our users to avoid issues when using `grpc-dotnet + Http.Resilience`.
Is it good to put here a link to the original issue we want to avoid? How do you think? Thank you.